scripts/windows/setup/IotEdgeSecurityDaemon.ps1

```diff
@@ -255,6 +270,21 @@ function Get-SecurityDaemon {
             Copy-Item "C:\ProgramData\iotedge\iotedged-windows\*" "C:\ProgramData\iotedge" -Force -Recurse
         }
 
+        foreach ($Name in "mgmt", "workload")
```

Comment:
This should only be done if `Test-UdsSupport` is true, right?

Reply:
Ah, yes correct. I'll update...